sdk/communication/azure-communication-administration/src/test/java/com/azure/communication/administration/PhoneNumberAsyncClientIntegrationTest.java

```diff
@@ -463,6 +465,29 @@ public void releasePhoneNumbersWithResponse() {
             .verifyComplete();
     }
 
+    @Test()
+    public void beginCreateSearch() {
+        List<String> phonePlanIds = new ArrayList<>();
+        phonePlanIds.add(PHONE_PLAN_ID);
+
+        CreateSearchOptions createSearchOptions = new CreateSearchOptions();
+        createSearchOptions
+            .setAreaCode(AREA_CODE_FOR_SEARCH)
+            .setDescription(SEARCH_OPTIONS_DESCRIPTION)
+            .setDisplayName(SEARCH_OPTIONS_NAME)
+            .setPhonePlanIds(phonePlanIds)
+            .setQuantity(2);
+
+        Duration duration = Duration.ofSeconds(1);
+        PhoneNumberAsyncClient client = this.getClient();
+        SyncPoller<PhoneNumberSearch, PhoneNumberSearch> res = 
+            client.beginCreateSearch(createSearchOptions, duration).getSyncPoller();
```

Comment:
If you call "getSyncPoller", your test is sync. Might as well call the sync client version and move this test to the sync client test file instead. 

For testing async version, you might want to test subscribe function of PollerFux with callback.